PICMI_Python/applied_fields.py

```diff
@@ -0,0 +1,118 @@
+"""Classes following the PICMI standard
+These should be the base classes for Python implementation of the PICMI standard
+"""
+import re
+
+from .base import _ClassWithInit
+
+# ---------------
+# Applied fields
+# ---------------
+
+
+class PICMI_ConstantAppliedField(_ClassWithInit):
+    """
+    Describes a constant applied field
+      - Ex: Constant Ex field (float) [V/m]
+      - Ey: Constant Ey field (float) [V/m]
+      - Ez: Constant Ez field (float) [V/m]
+      - Bx: Constant Bx field (float) [T]
+      - By: Constant By field (float) [T]
+      - Bz: Constant Bz field (float) [T]
+      - lower_bound=[None,None,None]: Lower bound of the region where the field is applied (vector) [m]
```

Comment:
Sorry for the possible confusion with what we discussed this morning.
I thought we were going to set the "width" of the fields in cells numbers.
(I have no preference) But just to check that I don't make mistakes, the way you wrote it is with position and length, right?

Reply:
Sorry for the confusion. I add this constant field just as an extra while I was implementing the analytic field. I also added a separate Mirror class which does has the width in meters and in the number of cells.